Sources/Component/CollapsibleView/HeaderView/DefaultCollapsibleHeaderView.swift

```diff
@@ -0,0 +1,72 @@
+//  Copyright © 2020 Jamit Labs GmbH. All rights reserved.
+
+import UIKit
+
+/// A stateful default header view for the collapsible view.
+///
+/// Example:
+/// ```swift
+/// let defaultCollapsibleHeaderView: DefaultCollapsibleHeaderView = .init()
+/// defaultCollapsibleHeaderView.model = .init(
+///     title: "Header title",
+///     titleFont: .systemFont(ofSize: 16.0),
+///     arrowImageUp: nil // TODO: Place an image here if needed
+/// )
+/// // TODO: adjust height of your header view.
+/// defaultCollapsibleHeaderView.heightAnchor.constraint(equalToConstant: 60.0).isActive = true
+/// ```
+public final class DefaultCollapsibleHeaderView: StatefulView<DefaultCollapsibleHeaderViewModel> {
+    private lazy var titleLabel: UILabel = {
```

Comment:
Drop closure. You can call `.init()` directly.

Reply:
Done.